components/popover/metadata/popover.yml

```diff
@@ -1,90 +1,606 @@
 name: Popover
 SpectrumSiteSlug: https://spectrum.adobe.com/page/popovers/
+sections:
+- name: Migration Guide
+  description: |
+    ### New Popover Positions
+    - Popover has 22 available positions. 10 positions use logical properties.
+    - Popover position and distance to source should be handled by implementation.
+    - In this document, Popover positions only impact default tip positioning.
+    - Default tip position is centered on the edge for top/bottom/left/right/start/end positions.
+    - Default tip position is distance from edge == popover corner radius for all other positions.
+    - Popover tip position can be overridden in implementation to center with the source by setting '--spectrum-popover-pointer-edge-offset' equal to half the width of the source (for top and bottom popovers) or half the height of the source (for side popovers).
+    - Popover tip SVG polygon can retain the same points for top and bottom (it is flipped in the CSS)
+    - Popover tip SVG polygon can retain the same points for left/right/start/end (it is flipped in the CSS)
+
 examples:
   - id: popover
     name: Standard
     markup: |
-      <div class="spectrum-Popover is-open">
-        <ul class="spectrum-Menu" role="listbox">
-          <li class="spectrum-Menu-item is-selected" role="option" aria-selected="true" tabindex="0">
-            <span class="spectrum-Menu-itemLabel">Ballard</span>
-          </li>
-          <li class="spectrum-Menu-item" role="option" tabindex="0">
-            <span class="spectrum-Menu-itemLabel">Fremont</span>
-          </li>
-          <li class="spectrum-Menu-item" role="option" tabindex="0">
-            <span class="spectrum-Menu-itemLabel">Greenwood</span>
-          </li>
-          <li class="spectrum-Menu-divider" role="separator"></li>
-          <li class="spectrum-Menu-item is-disabled" role="option" aria-disabled="true">
-            <span class="spectrum-Menu-itemLabel">United States of America</span>
-          </li>
-        </ul>
+      <div class="spectrum-Examples">
+        <div class="spectrum-Examples-item">
+          <h4 class="spectrum-Heading spectrum-Heading--sizeXS spectrum-Examples-itemHeading" style="margin-bottom: 16px;">Default</h4>
```

Comment:
I assume all those inline-margins had to be added so we can demo better the popover functionality. Ok with me :)

Reply:
The margins under the headings are just for a more comfortable display. Popover is absolutely positioned and has no set height and width, so without additional margin on the headings as well as the "dummy-spacing" elements, they would be invisible or overlapping with other elements in the document.